control-plane/subcommand/inject-connect/command.go

```diff
@@ -273,6 +279,12 @@ func (c *Command) init() {
 
 	c.flagSet.IntVar(&c.flagDefaultEnvoyProxyConcurrency, "default-envoy-proxy-concurrency", 2, "Default Envoy proxy concurrency.")
 
+	c.flagSet.BoolVar(&c.flagDefaultEnableConsulDataplaneAsSidecar, "default-enable-consul-dataplane-as-sidecar", false, "Default for enabling consul-dataplane as a sidecar container in the pod. ")
+	c.flagSet.IntVar(&c.flagDefaultSidecarProbeCheckInitialDelaySeconds, "default-sidecar-probe-check-initial-delay-seconds", 1, "Default number of seconds for the k8s initial delay before the readiness & liveness probe starts checking the consul-dataplane sidecar container.")
+	c.flagSet.IntVar(&c.flagDefaultSidecarProbePeriodSeconds, "default-sidecar-probe-period-seconds", 1, "Default number of seconds for the k8s period between startup probe checks.")
+	c.flagSet.IntVar(&c.flagDefaultSidecarProbeFailureThreshold, "default-sidecar-probe-failure-threshold", 10, "Default number of consecutive failures for the k8s startup probe before the consul-dataplane sidecar container is restarted.")
+	c.flagSet.IntVar(&c.flagDefaultSidecarProbeCheckTimeoutSeconds, "default-sidecar-probe-check-timeout-seconds", 5, "Default number of seconds for the k8s timeout for the startup probe checks.")
+
```

Comment:
Are the default value same as non sidecar method?

Reply:
Make sure it matches with last implementation. if yes, I dont have any concerns.